public/i18n/en.json

```diff
@@ -160,6 +161,7 @@
   "ftm-canada-province-ontario": "Ontario",
   "ftm-canada-manufacturer-call": "Federal call for manufacturers",
   "ftm-canada-supplier-call": "Federal call for suppliers",
+  "ftm-us-can-want-to-mail-link": "Our partners at <a target='_blank' rel='noreferrer noopener' href='https://www.mask-match.com'>https://www.mask-match.com</a> can help.",
```

Comment:
Can we use descriptive text here and drop the "https://" part?

**Current Line 164 to:**
`"ftm-us-can-want-to-mail-link": "Our partners at <a target='_blank' rel='noreferrer noopener' href='https://www.mask-match.com'>Mask Match</a> can help.",`